health_vara.py

```diff
@@ -105,6 +126,37 @@ def get_party_field_name(patient_field_name):
     return get_party_val(patient_field_name, 'party_field', patient_field_name)
 
 
+def datetime_sql_where_clause(original_op, value, column):
```

Comment:
When a date is given as a datetime input in Tryton, Tryton converts this to a datetime at the start of day.
So when using the UI from Germany, a date input like `14/08/2023` becomes `13/08/2023T22:00:00`

This function attempts to converts input into opinionated "more useful" search criteria.
For example, when someone types `<= 14/08/2023`, they don't mean, less than or equal to the **start** of the day, they mean less than or equal to the **end** of the day.

And when using `=`, they don't want to only get back results that match one exact utc timestamp, they mean any result that happened that day.